### ISSUE TITLE
Hg/Cd arc lines seem to be incomplete for MMT

### ISSUE THREAD

**Issue by palfeng** [pypeit/PypeIt#1352]
Hello, I'm Xinfeng, and have been reducing the MMT blue_Channel data using Pypeit v1.8.0. Everything works perfect except I found some of the Hg and Cd lines are missing in the linelist table: ~/anaconda3/envs/pypeit/lib/python3.8/site-packages/pypeit/data/arc_lines/lists/HgI_lines.dat and CdI_lines.dat. These lines are at the very blue end (which is the purpose of using MMT blue_channel), and without them the wavelength calibration is bad in these regions. The line list from MMT team is attached below. I have also checked that most of these lines are listed in NIST (and they are comparably strong), but somehow many of them are missing in Pypeit filles. 

Linelist provided by MMT blue_channel team for blue end:
3080.822000 Cd
3082.593000 Cd
3129.440000 HgIblend
3133.167000 Cd
3252.524000 Cd
3261.055000 Cd
3341.480000 Hg
3403.652000 Cd
3466.850000 CdIblend
3491.536000 ArII
3499.952000 Cd
3576.616000 ArII
3588.441000 ArII
3611.010000 CdIblend
3612.873000 Cd
3614.453000 Cd
3650.870000 HgIblend
3654.840000 Hg
3663.180000 HgIblend
3663.279000 Hg

**Comment by tbowers7** [comment]
Hi @palfeng.  Two of the lines you mentioned (and two others) were recently added, and will be available with v1.9.0 (note: all PypeIt wavelengths are in vacuum):
```
HgI |  2968.1495
HgI |  3022.384
CdI |  3134.0746
HgI |  3342.4450
```
As for most of your list, PypeIt tends to avoid blended lines or lines too close together because these lists are used by a variety of instruments at a variety of resolutions.  Blended or close lines at low resolution may affect centroiding and ultimately the quality of the wavelength calibration.

It may be possible to extend the (also new in v1.9.0) caching system to allow users to create and use their own linelists, but team discussion will be required.

**Comment by tbowers7** [comment]
Hi @palfeng. I've submitted a PR to allow users to specify their own line lists, and it should appear in the next release of PypeIt (probably v1.10.1).

A big caveat of using your own line lists is that the wavelengths MUST be **in vacuum** (this is contrary to IRAF and other line catalogs).  See the updated [Wavelength Calibration documentation](https://pypeit.readthedocs.io/en/develop/wave_calib.html) _after the above-mentioned PR merges_ for information about how to specify your additional lines.

**Comment by kbwestfall** [comment]
@tbowers7 Do you feel this is now addressed?  If so, please close the issue.  @palfeng , feel free to reopen or open a new issue if you still have concerns.

**Comment by palfeng** [comment]
Thanks! I think it’s good for now. I haven’t used Pypeit very recently, so I will try later and let you know if I get any issues.

-Xinfeng

**Comment by tbowers7** [comment]
> @tbowers7 Do you feel this is now addressed? If so, please close the issue. @palfeng , feel free to reopen or open a new issue if you still have concerns.

Yes, this was included in v11.0, so all set.
